mist-core/src/main/java/edu/snu/mist/core/task/groupaware/GroupAwareQueryManagerImpl.java

```diff
@@ -140,6 +140,9 @@ private GroupAwareQueryManagerImpl(final ScheduledExecutorServiceWrapper schedul
     this.applicationMap = applicationMap;
     this.groupMap = groupMap;
     this.checkpointPeriod = checkpointPeriod;
+    if (checkpointPeriod == 0) {
```

Comment:
How about adding this log message to `CheckpointManager`?

Reply:
Currently, the CheckpointManager does not have the field checkpointPeriod, as the period is related to generating MistCheckpointEvents. i.e., the CheckpointManager currently does not need to know the checkpointPeriod.

Do you think we should add the checkpointPeriod to the CheckpointManager, then add the logging message?

Reply:
This is because in our current implementation we don't do any actual checkpointing right now. We need to checkpoint regularly (which should be managed in `CheckpointManager`), so I think it is necessary to add `checkpointPeriod` in `CheckpointManager`.